### PR TITLE
Update ontology

### PR DESCRIPTION
**Pull request by elespdn**
The name of the branch is partially misleading: the branch contains some work on the text encoding, but the most important thing is that it contains the updated ontology.

### CODE REVIEW

**Comment by elespdn** [comment]
For what concerns the **DATA**:

1. the new version of the ontology will have conflicts with the two or three `:EstablishedText` that already exist in the data, because there have been a lot of changes to the properties of this class. The existing `:EstablishedText` can be deleted!

2. we can now change the permissions and open everything
```
PREFIX knora-base: <http://www.knora.org/ontology/knora-base#>
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
PREFIX knora-admin: <http://www.knora.org/ontology/knora-admin#>
with <http://www.knora.org/data/0112/roud-oeuvres>
delete {
    ?res knora-base:hasPermissions "CR knora-admin:ProjectAdmin|D knora-admin:ProjectMember" .
}
insert {
    ?res knora-base:hasPermissions "CR knora-admin:ProjectAdmin|D knora-admin:ProjectMember|V knora-admin:UnknownUser" .
}
where { 
    ?res knora-base:hasPermissions "CR knora-admin:ProjectAdmin|D knora-admin:ProjectMember" .
}
```

3. There are two other small changes to make to the data: replace "p.\d" with "p. \d" for pages of articles (insert whitespace after p.); and replace single 'p.' with 'pp.' when the reference is to multiple pages. The following sparql queries should do it (tested locally):

```
PREFIX knora-base: <http://www.knora.org/ontology/knora-base#>
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
PREFIX roud-oeuvres: <http://www.knora.org/ontology/0112/roud-oeuvres#>
with <http://www.knora.org/data/0112/roud-oeuvres>
delete {
    ?pages knora-base:valueHasString ?pagesString .
} 
insert {
    ?pages knora-base:valueHasString ?new_pagesString .
}
where { 
    ?resource roud-oeuvres:periodicalArticleIsInPages ?pages .
    ?pages knora-base:valueHasString ?pagesString .
    filter(regex(?pagesString, "p.(\\d)"))
    bind(replace(?pagesString, "p.(\\d)", "p. $1") AS ?new_pagesString)
}
```

```
PREFIX knora-base: <http://www.knora.org/ontology/knora-base#>
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
PREFIX roud-oeuvres: <http://www.knora.org/ontology/0112/roud-oeuvres#>
with <http://www.knora.org/data/0112/roud-oeuvres>
delete {
    ?pages knora-base:valueHasString ?pagesString .
} 
insert {
    ?pages knora-base:valueHasString ?new_pagesString .
}
where { 
    ?resource roud-oeuvres:periodicalArticleIsInPages ?pages .
    ?pages knora-base:valueHasString ?pagesString .
    filter(regex(?pagesString, "p. ([\\d]+-[\\d]+)"))
    bind(replace(?pagesString, "p. ([\\d]+-[\\d]+)", "pp. $1") AS ?new_pagesString)
}
```

**Comment by elespdn** [comment]
to summarize what's in the commits (some commits are too long ...)

closes #30, #33, #34, #35, #37, #38, #39, #40

**Comment by loicjaouen** [comment]
it worked fine on test, thanks for the well described steps

it is pushed on prod, please check that you find everything expect

**Comment by elespdn** [comment]
Yes, all fine, thanks!

**Comment by loicjaouen** [comment]
Cool :)

I am looking at the ontology file: there is a commit by @mrivoal (83f82deadd692d2b8a09d963f6f8b1f8c1ed39a7) that is not in this branch and should be imported so we can merge this PR

I should have seen that before :(
Let me make the changes and then you have a final look.
I'll add these later.

**Comment by loicjaouen** [comment]
whenever, you have time, please have a look at this last merge from `master` to `textEncoding`

If I am not mistaken, it should reflect the state of `textEncoding` as you expect it to be.

File: README.md [hunk 1]
```diff
@@ -1,35 +1,4 @@
 # oeuvres-roud
+Project specific ontologies and lists for the project "Oeuvres complètes de Gustave Roud".
 
-Project ["Gustave Roud, *Oeuvres complètes*"](https://www.unil.ch/clsr/home/menuinst/projets-de-recherche/gustave-roud-oeuvres-completes.html) (SNSF Project [157970](http://p3.snf.ch/Project-157970)).
```
Comment (elespdn) [inline README.md:3]: Test

**Review by elespdn — COMMENTED** [review]
Test, here general comment on the entire commit

File: README.md (hunk 1 above)
Comment (elespdn) [reply README.md:3]: Here comment on a specific line

**Review by loicjaouen — APPROVED** [review]
it looks good to me

File: README.md (hunk 1 above)
Comment (loicjaouen) [reply README.md]: c'est fait :)

**Comment by elespdn** [comment]
It looks good to me too

**Pull request merged**